test/reputation-system/client-calculations.js

```diff
@@ -74,8 +74,8 @@ process.env.SOLIDITY_COVERAGE
       });
 
       beforeEach(async () => {
-        await goodClient.resetDB();
         await goodClient.initialise(colonyNetwork.address);
+        await goodClient.resetDB();
```

Comment:
Why?

Reply:
Think this got caught in the crossfire of having to change the order of `initlialise` and `resetDB` elsewhere.